v0.9.0

PhileasL opened the pull request:
# v0.9.0 release !!!

Check [Readme](https://github.com/robotique-ecam/cdfr/blob/bdd18bb7ae2ae3b6fe0a34bba762ffe536066582/Readme.md) to know what has been done.

3wnbr1 commented:
Not ready for merge, `moves.json` with `.rbsd` files do not contain any translations

3wnbr1 commented on src/asterix/param/asterix.in.yml:
```diff
@@ -70,7 +70,7 @@ asterix:
     ros__parameters:
       use_sim_time: !Var use_sim_time
-
+      
```
No

3wnbr1 commented on src/modules/robot/param/robot.in.yml:
```diff
@@ -283,3 +278,7 @@ bt_navigator_rclcpp_node:
+
+map_server:
+  ros__parameters:
+    yaml_filename: dynamically_computed
```
1. Why yaml_filename doesn't end in .yaml ?
2. Why reordering ?

PhileasL replied:
It was previously [dynamically set](https://github.com/robotique-ecam/cdfr/blob/55829e3975926eb24a78ed10ebb0f6c0ef0dbf0e/src/modules/robot/robot/navigation_launch.py#L35) in navigation_launch, but i found a [better way](https://github.com/robotique-ecam/cdfr/blob/48711f59449e45ca7e9372d7a8120730cc54e6a5/src/modules/robot/robot/navigation_launch.py#L91) that keeps us from define "yaml_filemane" in robot.in.yaml.

PhileasL commented:
Still a bug in CI:
```bash
[lifecycle_manager-12] [INFO] [1618400912.890559907] [asterix.lifecycle_manager_navigation]: Configuring map_server
[map_server-11] [INFO] [1618400912.890893405] [asterix.map_server]: Configuring
[map_server-11] [INFO] [map_io]: Loading yaml file: /home/runner/work/cdfr/cdfr/install/map/share/map/map/map.yml
[map_server-11] [DEBUG] [map_io]: resolution: 0.02
[map_server-11] [DEBUG] [map_io]: origin[0]: -2.02
[map_server-11] [DEBUG] [map_io]: origin[1]: -2.02
[map_server-11] [DEBUG] [map_io]: origin[2]: 0
[map_server-11] [DEBUG] [map_io]: free_thresh: 0.196
[map_server-11] [DEBUG] [map_io]: occupied_thresh: 0.65
[map_server-11] [DEBUG] [map_io]: mode: trinary
[map_server-11] [DEBUG] [map_io]: negate: 0
[map_server-11] [INFO] [map_io]: Loading image_file: /home/runner/work/cdfr/cdfr/install/map/share/map/map/mapbleu.pgm
Error: rver-11] [ERROR] [map_io]: Failed to load image file /home/runner/work/cdfr/cdfr/install/map/share/map/map/mapbleu.pgm for reason: Magick: Unable to open file (/home/runner/work/cdfr/cdfr/install/map/share/map/map/mapbleu.pgm) reported by magick/blob.c:3058 (OpenBlob)
Error: rver-11] [ERROR] [1618400912.900290945] []: Caught exception in callback for transition 10
Error: rver-11] [ERROR] [1618400912.900308745] []: Original error: Failed to load map yaml file: /home/runner/work/cdfr/cdfr/install/map/share/map/map/map.yml
[map_server-11] [WARN] [1618400912.900327245] []: Error occurred while doing error handling.
[map_server-11] [FATAL] [1618400912.900337445] [asterix.map_server]: Lifecycle node map_server does not have error state implemented
Error: cle_manager-12] [ERROR] [1618400912.900867942] [asterix.lifecycle_manager_navigation]: Failed to change state for node: map_server
Error: cle_manager-12] [ERROR] [1618400912.900896142] [asterix.lifecycle_manager_navigation]: Failed to bring up all requested nodes. Aborting bringup.
```

Looks like the map properly load but there is something in CI that prevents the system from reading mapbleu.pgm.

Works well on tested computers, i don't know what this error means.

PhileasL commented:
Fixed. 
In fact while building map package, `execute_process(COMMAND python3 ${CMAKE_CURRENT_SOURCE_DIR}/map/map.py ${CMAKE_CURRENT_SOURCE_DIR})` in CMakeLists do not execute the python script, so mapbleu.pgm wasn't produced. 
I fixed it in [the last commit.](https://github.com/robotique-ecam/cdfr/pull/45/commits/5bbf89f771745083e70102a1ac7ef7e4d42ee4ed)
